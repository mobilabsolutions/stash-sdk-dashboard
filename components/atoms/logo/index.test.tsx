/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import { withTestSetup } from '../../../test_utils'
import Logo from './index'

const TestLogo = withTestSetup(Logo)

it('Logo Should Render', () => {
  const component = renderer.create(<TestLogo />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
