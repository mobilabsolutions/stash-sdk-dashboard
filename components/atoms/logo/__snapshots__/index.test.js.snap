// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Should Render 1`] = `
<img
  height={60}
  src="/static/images/logo.png"
/>
`;
